tests/static_data/single_publication/layers_test.py

```diff
@@ -47,8 +47,10 @@ def test_info(workspace, publ_type, publication):
     file_type = info_internal['file']['file_type']
     original_data_source = info.get('original_data_source', settings.EnumOriginalDataSource.FILE.value)
     item_keys = get_layer_info_keys(geodata_type=file_type, original_data_source=original_data_source)
+    info_keys = set(info.keys())
+    info_keys.discard('db_table')
```

Comment:
Consider putting 'db_table' to info_keys "(settings.GEODATA_TYPE_VECTOR, settings.EnumOriginalDataSource.FILE.value)" instead of this change